src/VisualStudio/Core/Def/Implementation/SymbolSearch/SymbolSearchContext.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.VisualStudio.Language.Intellisense.SymbolSearch;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.SymbolSearch
+{
+    internal class SymbolSearchContext : FindUsagesContext
+    {
+        internal SymbolSearchSource SymbolSource { get; }
+        internal SymbolOrigin LocalOrigin { get; }
```

Comment:
what is a SymbolOrigin?

Reply:
Declares that these results are exact results, and will be sorted above other results, e.g. remote or from metadata